core/src/test/java/org/infinispan/globalstate/GlobalStateTest.java

```diff
@@ -237,6 +240,37 @@ public void testCacheManagerNotifications(Method m) {
       }
    }
 
+   public void testNameSize(Method m) {
+      final StringBuilder cacheName = new StringBuilder();
+      for (int i = 0; i < 7; i++) {
+         cacheName.append(UUID.randomUUID());
+      }
+
+      while (cacheName.length() < 255) {
+         cacheName.append(" ÆÐþﺾ");
+      }
+
+      final String exceptionMessage = String.format("ISPN000663: Name must be less than 256 bytes, current name '%s' exceeds the size.", cacheName);
+      String state1 = tmpDirectory(this.getClass().getSimpleName(), m.getName());
+      GlobalConfigurationBuilder gcb = statefulGlobalBuilder(state1, true);
+      EmbeddedCacheManager cm = TestCacheManagerFactory.createClusteredCacheManager(false, gcb, null, new TransportFlags());
+
+      try {
+         cm.start();
+         expectException(exceptionMessage,
+               () -> cm.administration().getOrCreateCache(cacheName.toString(), new ConfigurationBuilder().build()),
```

Comment:
No need to create a new `Configuration` everytime, we can create it once and pass the same config to all of the test cases.